### PR TITLE
Docstring implementation on line.py

### PR DESCRIPTION
**Pull request by jremes-foss**
Hi.

Looks like there was one missing docstring on sympy/geometry/line.py. This has been now implemented with examples. 

In addition, I did some cleanup on the file. 

Thank you.

### CODE REVIEW

File: sympy/geometry/line.py [hunk 1]
```diff
@@ -158,7 +185,7 @@ def angle_between(l1, l2):
         return acos(v1.dot(v2)/(abs(v1)*abs(v2)))
 
     def arbitrary_point(self, parameter='t'):
-        """A parameterized point on the Line.
+        """A parametrized point on the Line.
```
Comment (smichr) [inline sympy/geometry/line.py]: I think the spelling was US-ok.

File: sympy/geometry/line.py [hunk 2]
```diff
@@ -203,9 +230,9 @@ def arbitrary_point(self, parameter='t'):
         t = _symbol(parameter)
         if t.name in (f.name for f in self.free_symbols):
             raise ValueError('Symbol %s already appears in object '
```
Comment (smichr) [inline sympy/geometry/line.py]: This is a good place to use filldedent:

```
raise ValueError(filldedent('''
    string
    ''' % t.name))
```

File: sympy/geometry/line.py [hunk 3]
```diff
@@ -1799,7 +1827,7 @@ def __new__(cls, p1, pt=None, slope=None, **kwargs):
                 p2 = Point(pt, dim=2)
             except (NotImplementedError, TypeError, ValueError):
                 raise ValueError('The 2nd argument was not a valid Point. '
-                'If it was a slope, enter it with keyword "slope".')
```
Comment (smichr) [inline sympy/geometry/line.py]: filldedent here, too

File: sympy/geometry/line.py [hunk 4]
```diff
@@ -2416,7 +2445,7 @@ def __new__(cls, p1, pt=None, direction_ratio=[], **kwargs):
                          p1.z + direction_ratio[2])
         else:
             raise ValueError('A 2nd Point or keyword "direction_ratio" must'
```
Comment (smichr) [inline sympy/geometry/line.py]: filldedent

**Comment by smichr** [comment]
I made a few comments. The rest looks good.

File: sympy/geometry/line.py [hunk 5]
```diff
@@ -1762,7 +1793,6 @@ class Line2D(LinearEntity2D, Line):
     Examples
     ========
 
-    >>> import sympy
     >>> from sympy import Point
     >>> from sympy.abc import L
     >>> from sympy.geometry import Line, Segment
```
Comment (gxyd) [inline sympy/geometry/line.py]: You can combine the imports as `from sympy.geometry import Point, Line, Segment`

Comment (smichr) [reply sympy/geometry/line.py]: Yes...this is rampant throughout the file but it wasn't introduced by this PR. I'm ok with this as is.

**Comment by smichr** [comment]
Thanks, this is in.

**Pull request merged**